src/token_request_handler.ts

```diff
@@ -46,6 +51,24 @@ export class BaseTokenRequestHandler implements TokenRequestHandler {
     return (response as TokenErrorJson).error === undefined;
   }
 
+  performRevokeTokenRequest(
+      configuration: AuthorizationServiceConfiguration,
+      request: RevokeTokenRequest): Promise<boolean> {
+    let revokeTokenResponse = this.requestor.xhr<any>({
+      url: configuration.revocationEndpoint + '?' + this.utils.stringify(request.toStringMap()),
+      dataType: 'json',  // adding implicit dataType
+      method: 'GET'
```

Comment:
And:
>  The client also includes its authentication credentials as described
>  in Section 2.3. of [RFC6749].

Not sure about that, for example the google idp does not require this, I was able to revoke token without an authorization header, gonna check what happens if I send it along anyway to be correct according rfc.